image/tools/lib/component/resources.sh

```diff
@@ -26,11 +26,16 @@ function backup_resource {
     local type=$1
     local ns=$2
     local dest=$3
-    local loop=$4
+    local loop=${4-default}
+
+    # Disable extended error checks. The check_resource function relies on a non-zero
+    # return code, which is interpreted as a failed command and causes the script to be
+    # terminated with the '-e' option
+    set +eo pipefail
```

Comment:
isn't this going to be overwritten by the entrypoint script anyway?

Reply:
the `+` is disabling the settings for the duration of this function. I enable them in the last line again.

Reply:
So this actually overrides the setting in entrypoint for this function